core/templates/dev/head/pages/collection_editor/settings_tab/collection_permissions_directive.html

```diff
@@ -0,0 +1,31 @@
+<script type="text/ng-template" id="inline/collection_permissions_directive">
+  <div role="form" class="form-horizontal">
+    <div class="row">
+      <div class="col-lg-5 col-md-5 col-sm-5">
+        <h3>Managers</h3>
+        <ul>
+          <li ng-repeat="ownerName in ownerNames track by $index">
+            <[ownerName]>
+          </li>
+        </ul>
+      </div>
+
+
+      <div class="col-lg-7 col-md-7 col-sm-7">
+        <h3>Permissions</h3>
+        <div ng-if="isPrivate">
+          This collection is <strong>private</strong>. Anyone with the link can access it.
```

Comment:
Drop "Anyone with the link can access it." -- I don't think this is intended long-term.